pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -102,21 +102,143 @@ type ClusterWorkspaceType struct {
 
 	// +optional
 	Spec ClusterWorkspaceTypeSpec `json:"spec,omitempty"`
+
+	// +optional
+	Status ClusterWorkspaceTypeStatus `json:"status"`
 }
 
+const (
+	// ClusterWorkspaceCreatorUserLabel is the name of the user creating this ClusterWorkspace.
+	// It is used by the initialization virtual workspace to authorize object creation. It is applied during
+	// admission, removed after initialization and is not exposed to the user in Workspaces.
+	ClusterWorkspaceCreatorUserLabel = "tenancy.kcp.dev/creator-user"
+	// ClusterWorkspaceCreatorGroupsLabel is a comma separated list of group of the user creating
+	// this ClusterWorkspace. It is used by the initialization virtual workspace to authorize object creation
+	// It is applied during admission, removed after initialization and is not exposed to the
+	// user in Workspaces.
+	ClusterWorkspaceCreatorGroupsLabel = "tenancy.kcp.dev/creator-groups"
+)
+
 type ClusterWorkspaceTypeSpec struct {
-	// initializers are set of a ClusterWorkspace on creation and must be
+	// bootstrapManifests references a set of Kubernetes manifests that are applied to the
+	// workspace on initialisation.
+	//
+	// If a workspace type extends on at a higher level, the bootstrap manifests of
+	// the higher level are applied first, then those of the lower levels.
```

Comment:
Do we have any learnings (perhaps from CVO?) about what sort of flexibility we need in ordering here? Do we expect this to grow to be generic manifests?

Reply:
just pure creation without templating. For anything advanced we have initializers. Would try to keep out of templating business for sure.